wolfssl/wolfcrypt/asn_public.h

```diff
@@ -443,9 +443,10 @@ typedef struct Cert {
     CertExtension customCertExt[NUM_CUSTOM_EXT];
     int customCertExtCount;
 #endif
-    void*   decodedCert;    /* internal DecodedCert allocated from heap */
-    byte*   der;            /* Pointer to buffer of current DecodedCert cache */
-    void*   heap;           /* heap hint */
+    void*   decodedCert;        /* internal DecodedCert allocated from heap */
+    byte*   der;                /* Pointer to buffer of current DecodedCert cache */
+    void*   heap;               /* heap hint */
+    byte    extBasicConstSet:1; /* Extended Basic Constraint Set */
```

Comment:
Did you mean extension rather than extended?

Reply:
I believe `ext` in the Certs and DecodedCerts denotes extended

Reply:
@dgarske can you weigh in? Pretty sure the only time it means extended is extended key usage, which is confusing because extended key usage is also an extension.

Reply:
Can we match the x509 name `basicConstSet`?

Reply:
Matched with x509 name `basicConstSet`.